Read.pm

```diff
@@ -903,6 +901,11 @@ sub ReadData {
 	    $debug and print STDERR "Opening \U$1\E $txt $using\n";
 	    $sxc = Spreadsheet::ReadSXC::read_sxc      ($txt, $sxc_options)	or  return;
 	    }
+	# treat all refs as a filehandle
```

Comment:
And this can stay as is, as the check above would have croak'ed on older versions